src/data/template.ts

```diff
@@ -1 +1,18 @@
-export const isDynamicTemplate = (value: string) => value.includes("{{");
+const isTemplateRegex = new RegExp("{%|{{|{#");
+
+export const isTemplate = (value: string): boolean =>
+  isTemplateRegex.test(value);
+
+export const hasTemplate = (value: unknown): boolean => {
```

Comment:
Shouldn't we make a specific function just for objects to differentiate from `isTemplate`? 
```suggestion
export const hasTemplate = (value: Record<string, unknown>): boolean => {
```